src/modules/core/components/ColorSelect/ColorSelect.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useMemo, ReactNode, ComponentProps } from 'react';
+import { defineMessages } from 'react-intl';
+
+import { Select, SelectOption, Form } from '~core/Fields';
+import ColorTag, { Color } from '~core/ColorTag';
+
+import styles from './ColorSelect.css';
+
+const MSG = defineMessages({
+  labelColorSelect: {
+    id: 'ColorSelect.labelColorSelect',
+    defaultMessage: 'Select color',
+  },
+});
+
+interface FormValues {
+  activeColor: Color;
+}
+
+interface Props {
+  /** Should `select` be disabled */
+  disabled?: boolean;
+
+  /** Active color */
+  activeOption: Color;
+
+  /** Callback function, called after value is changed */
+  onColorChange?: (color: Color) => any;
+}
+
+const displayName = 'ColorSelect';
+
+const ColorSelect = ({ disabled, activeOption, onColorChange }: Props) => {
+  const onChange = useCallback(
+    (color: Color) => {
+      if (onColorChange) {
+        return onColorChange(color);
+      }
+      return null;
+    },
+    [onColorChange],
+  );
+
+  const renderActiveOption = useCallback<
+    (option: SelectOption | undefined, label: string) => ReactNode
+  >(() => {
+    return <ColorTag color={activeOption || Color.Yellow} />;
+  }, [activeOption]);
+
+  const options = useMemo<ComponentProps<typeof Select>['options']>(() => {
+    const colors = Object.keys(Color).filter(
+      (val) => typeof Color[val as any] === 'number',
+    );
+    return [
+      ...colors.map((color) => {
+        return {
+          children: <ColorTag color={Color[color]} />,
+          label: `${color}`,
+          value: `${Color[color]}`,
+        };
+      }),
+    ];
+  }, []);
+
+  return (
+    <Form<FormValues>
+      initialValues={{
+        activeColor: activeOption || Color.Yellow,
+      }}
+      onSubmit={() => {}}
+    >
+      <div className={styles.wrapper}>
```

Comment:
Usually, we have the first "wrapper" class in the component named `main`.

Please try to do this from now on, for consistency's sake.